session_1/hello_world/app/views/page/about_me.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Welcome To IRIS Boot Camp</h1>
+<p>Introduction to RoR</p>
+<body> In a nut shell :p </body>
```

Comment:
the contents of this file are automatically nested inside a body tag.
Hence using a body tag again is a bit redundant, for text try using a <p> tag instead😅